Services/Help/classes/class.ilObjHelpSettings.php

```diff
@@ -71,13 +71,13 @@ public static function writeHelpModuleLmId(
     public function uploadHelpModule(
         array $a_file
     ) : void {
-        $id = $this->createHelpModule();
+        $id = self::createHelpModule();
         
         try {
             $imp = new ilImport();
             $conf = $imp->getConfig("Services/Help");
             $conf->setModuleId($id);
-            $new_id = $imp->importObject("", $a_file["tmp_name"], $a_file["name"], "lm", "Modules/LearningModule");
+            $new_id = $imp->importObject(""/* TODO PHP8-REVIEW This must be NULl or an object */, $a_file["tmp_name"], $a_file["name"], "lm", "Modules/LearningModule"); //
```

Comment:
I would pass `null` here, but I am not sure.

Reply:
Yes, definitely. This parameter seems to be obsolete anyway, I will have to check the import component...